src/current/advisories/a104309.md

```diff
@@ -80,6 +80,32 @@ If data is found, it can be re-emitted in two ways:
 - By restarting the changefeed with `initial_scan` set to `yes` and without a cursor timestamp. This will perform a full export of all current data across the changefeed.
 - By restarting the changefeed and providing a cursor timestamp. If a cursor timestamp is provided, an initial scan will not be performed. Instead, updates above the cursor timestamp will be emitted. The cursor timestamp can be initialized below the timestamp of omitted writes (refer to the reference documentation for [CREATE CHANGEFEED](https://www.cockroachlabs.com/docs/stable/create-changefeed)). However, specifying a cursor timestamp farther in the past than the [`gc.ttlseconds`](https://www.cockroachlabs.com/docs/stable/configure-replication-zones#gc-ttlseconds) replication zone variable, which defaults to `14400` seconds (4 hours), results in an error.
 
+{{site.data.alerts.callout_info}}
+This fix introduces a [bug](https://github.com/cockroachdb/cockroach/issues/119536) that could cause a rangefeed’s resolved timestamp to stop advancing. The corresponding changefeed will appear to be stalled in `RUNNING` state in certain conditions: If a rangefeed is running on a follower on a recently-merged range and encounters an aborted transaction, then the resolved timestamp may stall if the node has a stale range descriptor cache. Events will still be emitted as normal.
```

Comment:
```suggestion
This fix introduces a [bug](https://github.com/cockroachdb/cockroach/issues/119536) that could cause a rangefeed’s resolved timestamp to stop advancing. The corresponding changefeed will appear to be stalled in `RUNNING` state in certain conditions: If a rangefeed is running on a follower on a recently-merged range and encounters an aborted transaction, then the resolved timestamp may stall if the node has a stale range descriptor cache. Events like row updates will still be emitted as normal, only checkpoints will stall.
```

Reply:
"If a rangefeed is running on a follower on a recently-merged range, the node has a stale range descriptor cache, and the rangefeed encounters an aborted transaction, then the resolved timestamp may stall. Events such as row updates will still be emitted as normal; new checkpoints will not be emitted."

Reply:
Edited Erik's suggestion to add these changes.